src/litserve/server.py

```diff
@@ -390,17 +389,22 @@ async def stream_predict(request: self.request_type) -> self.response_type:
                 self.app.add_middleware(middleware)
 
     @staticmethod
-    def generate_client_file():
+    def generate_client_file(port: Union[str, int] = 8000):
         src_path = os.path.join(os.path.dirname(__file__), "python_client.py")
         dest_path = os.path.join(os.getcwd(), "client.py")
 
-        if os.path.exists(dest_path):
```

Comment:
@aniketmaurya Would you agree we should remove this? I think it makes sense to overwrite the template file each time the server is spun up, especially now that we make the port dynamic (case: user changes port number in `Server.run()`)

Reply:
client.py is still a static template. If an user has their client.py then we should never replace it.